modules/rosa-attaching-the-policy.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * rosa-adding-additional-constraints-for-ip-based-aws-role-assumption/rosa-attaching-the-policy.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-attaching-the-policy_{context}"]
+= Attaching the Identity-based IAM policy
+
+:context: rosa-attaching-the-policy
+
+toc::[]
+
+Once you have created an Identity-based IAM policy, you need to attach it to the relevant IAM users, groups, or roles in your AWS account. This enforces IP-based role assumption prevention for those entities.
+
+.Procedure
+
+. Navigate to the IAM console in the AWS Management Console.
+. Select the IAM user, group, or role to which you want to attach the policy.
+. In the *permissions* tab, click *Add inline policy* or *Attach existing policies*.
```

Comment:
On AWS console,the tab and button names are different now, they are 'Permissions', 'Add Permissions' and 'Create inline policy' in the dropdown box of 'Add Permissions'.